compiler/passes/convert-uast.cpp

```diff
@@ -4384,16 +4389,11 @@ bool LoopAttributeInfo::insertBlockSizeCall(Converter& converter, BlockStmt* bod
   static int counter = 0;
 
   if (blockSizeAttr) {
-    if (blockSizeAttr->numActuals() != 1) {
-      USR_FATAL(blockSizeAttr->id(),
-                "'@gpu.blockSize' attribute must have exactly one argument: "
-                "the block size");
```

Comment:
Do we have this check elsewhere now? I may have missed it.

Reply:
Do we have a test for it? If not, maybe we should (ok to add in this PR).

Reply:
NVM. I see where you added the check, and can see that as a benefit of the function-based vs primitive-based approach. But, I'd still like to hear from you other benefits if any.